quantum/quantum_keycodes.h

```diff
@@ -385,113 +385,114 @@ enum quantum_keycodes {
     UNICODE_MODE_WIN,     // 5CE4
     UNICODE_MODE_BSD,     // 5CE5
     UNICODE_MODE_WINC,    // 5CE6
+    UNICODE_MODE_EMACS,   // SCE7
```

Comment:
Keycodes cannot be added to the middle of the list as it breaks existing assumptions in things like via

Reply:
OK I can change that

Reply:
Now exists twice, along with all the comment changes.

Reply:
Drat. Careless fix

Reply:
This has been fixed now. I had ran `git checkout`  from the wrong source.